classes/file/FileManager.inc.php

```diff
@@ -234,7 +234,6 @@ function readFileFromPath($filePath, $output = false) {
 	function downloadFile($filePath, $mediaType = null, $inline = false, $fileName = null) {
 		$result = null;
 		if (HookRegistry::call('FileManager::downloadFile', array(&$filePath, &$mediaType, &$inline, &$result, &$fileName))) return $result;
-		$postDownloadHookList = array('FileManager::downloadFileFinished', 'UsageEventPlugin::getUsageEvent');
```

Comment:
I see the FileManager::downloadFileFinished is still there, at the end of the function, but what about UsageEventPlugin::getUsageEvent ?